README.md

```diff
@@ -35,7 +35,7 @@ To integrate `GliaWidgets` into your project via [CocoaPods](https://cocoapods.o
 source 'https://github.com/CocoaPods/Specs.git'
 source 'https://github.com/salemove/glia-ios-podspecs.git'
```

Comment:
we could drop this line also since our SDK is available in the official Cocoapods repo...

Reply:
Fixed https://github.com/salemove/ios-sdk-widgets/pull/436/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L36